DragaliaAPI.Test/Integration/Dragalia/DragonControllerTest.cs

```diff
@@ -0,0 +1,578 @@
+﻿using DragaliaAPI.Controllers.Dragalia;
```

Comment:
Can you rename this file/class to DragonTest to keep with the naming scheme of integration tests? xxxControllerTest is normally used for unit tests of controllers